### PR TITLE
Add processor for box-shadow

### LINKED ISSUE #6: Processor for adding shadow
## Description

Some elements will need a shadow, and this should be configurable from Gutenberg. We are planning to add a class `has-shadow` to the elements where we want to apply it.

It could be applied inside any tag.

## Style
Each element with `has-shadow` should have these properties
```
box-shadow: 0 4px 14px 0 rgba(31,56,197,0.09), 0 2px 4px 0 rgba(31,56,197,0.12);
```

## Where to check
Inside `/common-styles/` page -> Div with shadow
Invision Homepage -> FAQ

### PR DESCRIPTION
**Pull request by iamuchejude**
Fixes #6

### CODE REVIEW

File: packages/frontity-org-theme/src/processors/box-shadow.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { css } from "frontity";
+
+const boxShadowProcessor = {
+  name: "box-shadow",
+  test: node =>
```
Comment (luisherranz) [inline packages/frontity-org-theme/src/processors/box-shadow.ts]: `test` has changed as well to maintain consistency:
```js
test: ({ node }) => 
```

File: packages/frontity-org-theme/src/index.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import FrontityOrg from "../types";
 import Theme from "./components";
-import { backgroundColor, textColor } from "./processors";
+import { boxShadow , backgroundColor, textColor } from "../src/processors";
```
Comment (SantosGuillamot) [inline packages/frontity-org-theme/src/index.ts]: Why do we change here the folder? Shouldn't it be `"./processors"`?

Comment (iamuchejude) [reply packages/frontity-org-theme/src/index.ts]: I guess all of the code (except some configs) that powers that applications should be in the source `src` folder.

Comment (luisherranz) [reply packages/frontity-org-theme/src/index.ts]: > I guess all of the code (except some configs) that powers that applications should be in the source src folder.

You are right about that, but that's already the case. The imports `"./processors"` and `"../src/processors"` are equivalent.

Please note that with `..` you are going up a folder, so you need to indicate `/src` to come back to the same folder where you are. 
On the other hand, `.` just means "the same folder where you are", so you end up in the same place in both cases.

If this is not clear let me know and I'll give you a more detailed explanation.

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 45c5139997f5cf53a70b181d2b9719960a3e8b70

https://frontity-org-b3ybeq91l.now.sh

**Pull request merged**